### ISSUE TITLE
Master branch No April or October

### ISSUE THREAD

**Issue by mikewright14** [eternicode/bootstrap-datepicker#1533]
In the master branch March appears twice (03/04) this throws all the other months out of sync (july being an 08 month). October is completely missing. 1.4 Release is fine for this issue, so there is a translation between the 2 releases.

**Comment by hebbet** [comment]
can you provide an example for that?

**Comment by grimmlink** [comment]
Hi,

I have the same problem, update the plugin today and having this : 
![image](https://cloud.githubusercontent.com/assets/3241292/9521188/a24fc7d6-4ccd-11e5-8726-e59cb45939b7.png)


here is the code : 
```javascript
            $(this).datepicker({
                weekStart: 1,
                todayBtn: "linked",
                autoclose: true,
                todayHighlight: true
            });
```

**Comment by mikewright14** [comment]
Yeah 1.4.0 is fine however Master has the error.

**Comment by jameskirsop** [comment]
Same problem here, too. The following instantiation yields incorrect month names displayed on all months. The actual calendar looks OK (correct pairing of day to day of the week on the picker).
If you require some further information/details, let me know!

    .datepicker({
        format: 'mm/dd/yyyy',
        autoclose: true,
        startdate: new Date()
      });

**Comment by srg-kas** [comment]
Have just downloaded last release, it didnt help, found the "fill" function and added 1 to the month in Date creating, for me it's working now. No aftermath found, tell me if I doing wrong.

**Comment by hebbet** [comment]
@srg-kas 1.4 or 1.5?

**Comment by srg-kas** [comment]
@hebbet took from master branch.

**Comment by srg-kas** [comment]
@hebbet but from dist directory, not from js.

**Comment by hebbet** [comment]
master branch is currently broken. So use 1.4

**Comment by srg-kas** [comment]
@hebbet yeah, thanks, everything works well in 1.4.

**Comment by acrobat** [comment]
Can you guys verify if this issue is resolved in the current master, PR #1503 possibly fixed this issue

**Comment by grimmlink** [comment]
looks fine to me

**Comment by acrobat** [comment]
Thanks for the feedback @grimmlink!

Fixed in #1503
